services/workflows-service/package.json

```diff
@@ -30,6 +30,8 @@
     "db:reset:dev:with-data": "npm run db:reset:dev && npm run db:data-migration:migrate && npm run db:data-sync",
     "db:init": "npm run db:migrate-dev -- --name 'initial version' && npm run db:migrate-up seed",
     "prisma:generate": "prisma generate",
+    "docker:redis": "docker compose -f docker-compose.redis.yml up -d --wait",
```

Comment:
If you run this command this way, the `REDIS_PORT` won't be loaded from the dotenv. Is it part of the requirements?

Reply:
from what i checked it worked